packages/amplify_auth_cognito/test/amplify_auth_cognito_confirmSignIn_test.dart

```diff
@@ -19,6 +19,23 @@ import 'package:amplify_auth_cognito/amplify_auth_cognito.dart';
 import 'package:amplify_auth_cognito/method_channel_auth_cognito.dart';
 import 'package:amplify_core/amplify_core.dart';
 
+extension IsEqual on SignInResult {
+  bool isMostlyEqual(SignInResult comparator) {
+    if(
+      comparator.isSignedIn == isSignedIn &&
+      comparator.nextStep.signInStep == nextStep.signInStep &&
+      comparator.nextStep.additionalInfo.length == nextStep.additionalInfo.length &&
+      comparator.nextStep.codeDeliveryDetails.destination == nextStep.codeDeliveryDetails.destination &&
+      comparator.nextStep.codeDeliveryDetails.attributeName == nextStep.codeDeliveryDetails.attributeName &&
+      comparator.nextStep.codeDeliveryDetails.deliveryMedium == nextStep.codeDeliveryDetails.deliveryMedium
```

Comment:
nit: you can just return this statement.

Reply:
ack, yeah good call.